specs/003-implementa-todo-lo/contracts/auth-api.yaml

```
@@ -0,0 +1,497 @@
+openapi: 3.0.3
+info:
+  title: Flash Cards Authentication API
+  description: API contracts for user authentication and management
+  version: 1.0.0
+  contact:
+    name: Flash Cards Auth API
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)